packages/react/.storybook/addon-fathom/register.js

```diff
@@ -0,0 +1,27 @@
+import { window } from 'global';
+import { addons } from '@storybook/addons';
+import { STORY_CHANGED } from '@storybook/core-events';
+
+addons.register('fathom', api => {
+  (function(f, a, t, h, o, m) {
+    a[h] =
+      a[h] ||
+      function() {
+        (a[h].q = a[h].q || []).push(arguments);
+      };
+    (o = f.createElement('script')), (m = f.getElementsByTagName('script')[0]);
+    o.async = 1;
+    o.src = t;
+    o.id = 'fathom-script';
+    m.parentNode.insertBefore(o, m);
+  })(document, window, 'https://cdn.usefathom.com/tracker.js', 'fathom');
+  fathom('set', 'siteId', 'LLBKOSZK');
```

Comment:
Could we support the "pushState Navigation" example potentially? https://usefathom.com/support/tracking

```js
fathom('set', 'spa', 'pushstate');
```

Reply:
I had that but it still did not seem to register the correct paths...